pokedex/data/csv/generation_names.csv

```diff
@@ -1,19 +1,43 @@
 generation_id,local_language_id,name
+1,1,だいいっせだい
+1,2,daiissedai
```

Comment:
Language 2 is for the "official roomaji" translations of pokemon names. It isn't necessary here.

Reply:
Thanks for letting me know! I saw some fields that had it and others that did not so that rule is really helpful to know. I've removed the roomaji fields in a new commit.